src/usr/local/www/widgets/widgets/thermal_sensors.widget.php

```diff
@@ -136,9 +136,9 @@ function getBoolValueFromConfig(&$configArray, $valueKey, $defaultValue, $widget
 	//handle checkboxes separately
 	saveGraphDisplaySettings($user_settings, $_POST, "thermal_sensors_widget_show_raw_output");
 	saveGraphDisplaySettings($user_settings, $_POST, "thermal_sensors_widget_show_full_sensor_name");
-	saveGraphDisplaySettings($user_settings, $_POST, "thermal_sensors_widget_pulsate_warning");
-	saveGraphDisplaySettings($user_settings, $_POST, "thermal_sensors_widget_pulsate_critical");
 	saveGraphDisplaySettings($user_settings, $_POST, "thermal_sensors_widget_show_fahrenheit");
+	//saveGraphDisplaySettings($user_settings, $_POST, "thermal_sensors_widget_pulsate_warning");
+	//saveGraphDisplaySettings($user_settings, $_POST, "thermal_sensors_widget_pulsate_critical");
```

Comment:
Remove the code instead of commenting it out and change the commit message to:
```
Remove old code references in thermal widget

Originally removed in https://redmine.pfsense.org/issues/5369
```

Reply:
Removed and ammended the commit.